README.md

```diff
@@ -4,13 +4,17 @@
 
 Run [DirectAI](https://directai.io) on a [LiveKit](https://livekit.io) stream and see the results in real time!
 
-We will open a browser using LiveKit's [meet](https://meet.livekit.io) product and have DirectAI join the room! We'll then build on-the-fly detectors matching your descriptions and track objects in the video feed in real time!
+We will open a browser using LiveKit's [meet](https://meet.livekit.io) product and have DirectAI join the room! We'll then build on-the-fly object detectors matching your descriptions and track objects in the video feed in real time!
 
 ## Getting Started
 - You'll need DirectAI credentials to use the API - generate those by heading to our [landing page](https://directai.io) and logging in/signing up via the "Get API Access" button. New users get their first **10k streaming images free**!
 - Generate two LiveKit tokens for the same room, one for a bot powered by DirectAI and one for your user. Check out Liveit's instructions on [Generating Keys & Tokens](https://docs.livekit.io/cloud/keys-and-tokens/). Note that you can also generate them via [Livekit's CLI](https://github.com/livekit/livekit-cli).
+- Optionally, create a third token for the same room, which will be used to receive real-time results via the WebRTC data channel. (Results can also be received via a webhook.)
```

Comment:
to clarify: the data channel requires a separate livekit connection with its own set of credentials. this connection _does not_ generate a third room attendee

Reply:
this connection does create a third room attendee, which just listens for events. it's akin to having someone join google meets with no video or audio, just looking at the chat

Reply:
I was under the impression that there was a way for a token to be issued s.t. the "process" monitoring that room wouldn't be visible to Livekit viewers. Having an extra attendee with a dark screen isn't great from the user experience perspective - I'm willing to address in a different PR but wanted to point it out

Reply:
yeah, that's controlled on the token generation side, not our side .. there's a checkbox they can check that will make the person invisible when they issue the token

Reply:
i'd like to make a note in the docs for this. again, can be done later (#3)